config/settings/base.py

```diff
@@ -242,6 +242,9 @@
 DEFAULT_FROM_EMAIL = env(
     "DJANGO_DEFAULT_FROM_EMAIL", default="MuckRock <info@muckrock.com>"
 )
+PRESSPASS_FROM_EMAIL = env(
+    "DJANGO_DEFAULT_FROM_EMAIL", default="MuckRock <info@muckrock.com>"
```

Comment:
This should be something different from the default, right? We'll need to support a default (MuckRock) and PressPass in the same environment.

Reply:
doh! copy/paste (user) error. I'll fix this.